packages/editor/src/components/global-keyboard-shortcuts/visual-editor-shortcuts.js

```diff
@@ -161,12 +161,16 @@ const EnhancedVisualEditorGlobalKeyboardShortcuts = compose( [
 		};
 	} ),
 	withDispatch( ( dispatch ) => {
+		// This component should probably be split into to
+		// A block editor specific one and a post editor one.
```

Comment:
What's the reasoning for this? Is it because of the undo/redo in the component?

Reply:
yes, ideally a user of the block editor module should have the block shortcuts enabled by default and would be able to add other shortcuts if needed (undo/redo in that case)

Reply:
Nit: there is `A` in the middle of the sentence. However more importantly, do we want to add `TODO:` items for all comments introduced in this PR? If yes, ideally I would see them with the referenced issue to make it easier to remove later.